<launch>


    <node name="kittiAdapter" pkg="fusion_localization_zc" type="kittiAdapter" output="screen">
        <param name="dataset_folder" type="string" value="/media/zhouchang/data/kitti/raw_data/2011_09_30_drive_0027_extract" />
        <!--param name="sequence_number" type="string" value="00" /-->
        <param name="calib_folder" type="string" value="/media/zhouchang/data/kitti/raw_data/2011_09_30_drive_0027_extract" />
        <param name="to_bag" type="bool" value="false" />
        <param name="output_bag_file" type="string" value=" $(find fusion_localization)/tmp/kitti.bag" /> <!-- replace with your output folder -->
        <param name="publish_delay" type="int" value="1" />
    </node>
</launch>